.travis.yml

```diff
@@ -1,20 +1,27 @@
 language: python
 python:
 - '2.7'
-env:
-- ES_HOST="search-dos-azul-test-a998691e-uuzze4esxbslgnc4i3pnbzjpq4.us-west-2.es.amazonaws.com" TEST_ES_DOMAIN="dos-azul-test-a998691e"
 branches:
   only:
   - master
 before_install:
 - sudo apt-get update -qq
 - pip install -r dev-requirements.txt
 - pip install -r requirements.txt
+# We set these environment variables here instead of in `env` as
+# provision/provision.py requires boto3
+- export TEST_ES_DOMAIN=$(python provision/provision.py setup)
+- echo ${TEST_ES_DOMAIN}
+# We add the `:7` to strip the leading `http://`
+- export ES_HOST=$(python provision/provision.py get-endpoint ${TEST_ES_DOMAIN})
+- export ES_HOST=${ES_HOST:7}
```

Comment:
It's good that this is documented! Is there ever a change this could somehow switch to https and mess things up?

Reply:
That is true. I'll push a fix before I merge.